spec/javascripts/app/views/help_view_spec.js

```diff
@@ -134,4 +141,28 @@ describe("app.views.Help", function(){
       });
     });
   });
+
+  describe("chat section", function(){
+    describe("chat enabled", function(){
+      beforeEach(function(){
+        gon.chatEnabled = true;
```

Comment:
Ok. I thought chat `this.chat_enabled` was just an alias to the function `chatEnabled()` and was called each time the page is rendered.